packages/application-shell-connectors/src/components/project-extension-image-regex/project-extension-image-regex.spec.tsx

```diff
@@ -1,5 +1,5 @@
+//@ts-nocheck
```

Comment:
I was wondering why do we need this TS ignore directive?

Reply:
Taking another look, I guess it could be because of the typing of the test components.
If that's the case, I think adjusting the `WrappedTestComponent` like this could help:
```
const WrappedTestComponent = withProjectExtensionImageRegex<TBasicTestComponentProps>()(BasicTestComponent) as ComponentType;
```

Reply:
Oh its a temporary holder, I was going to clean it up initially